src/entity/entity.h

```diff
@@ -134,6 +136,42 @@ EntityID EntitySystem::add(E entity) {
     return id;
 }
 
+using FieldNameType = const char *;
+///*
+// Returns true if the given entity type has a field with the
+// given name, note that this name has to be a FieldName::.
+// TODO(ed): FieldName:: should be enforced via a typecheck.
```

Comment:
```suggestion
using FieldNameType = const char *;
// TODO(ed): FieldName:: should be enforced via a typecheck.
///*
// Returns true if the given entity type has a field with the
// given name. Note that this name has to be a <code>FieldName::</code>.
```

Reply:
The diff isn't that readable. There's two changes here:
1. Moving the TODO to the top (so it isn't part of the documentation)
2. ", note" => ". Note"
3. `<code>`